Build/mk_scripts/mk_make

```diff
@@ -77,12 +84,12 @@ $FC --version
 NCEPLIBS="../../libFMS/${COMPILER}/${bit}/libFMS.a"
 if [ ${CONFIG} = 'shield' ] ; then
   NCEPLIBS+=" ./libgfs.a"
-  NCEPLIBS+=" -L../../nceplibs/${COMPILER}/ -lbacio_4  -lsp_v2.0.2_d  -lw3emc_d  -lw3nco_d"
+  NCEPLIBS+=" -L../../nceplibs/${COMPILER}/ -lbacio  -lsp_d  -lw3emc_d  -lw3nco_d"
```

Comment:
You could set a library path variable in Build/COMPILE that would point to internal or external locations and simply refer to it here.  It would save you from managing the various links.

Reply:
I created a new variable "NCEP_DIR" that if internal libraries are used is set to SHiELD_build/build, and if external libraries are used is set to the value of EXTERNAL_LIBS and then I use that variable here in mk_make.  I also no longer link the external library location to SHiELD_build/Build and now in COMPILE after libFMS is build and NCEP_DIR is set, I verify that all the necessary libraries exist before moving onto the SHiELD build.  Please let me know if you had something else in mind by this comment.